### ISSUE TITLE
Front matter alignment

### ISSUE THREAD

**Issue by Bebotron** [miligithub/MinesThesisTemplate#5]
Text in abstract, signatures, acknowledgements, and dedications are justified, but they should be left aligned to be consistent with the rest of the thesis text.

**Comment by miligithub** [comment]
Yes, currently we only do left-aligned after front matter. Try move this [line](https://github.com/miligithub/MinesThesisTemplate/blob/5c7c648b41ce2fb270cb16cfaf8501e66147cd1d/mines-thesis.cls#L306) up in the [front matter section](https://github.com/miligithub/MinesThesisTemplate/blob/5c7c648b41ce2fb270cb16cfaf8501e66147cd1d/mines-thesis.cls#L291-L307). I did not test it, so might need to do small fixes per page.
